lib/helper/storage.js

```diff
@@ -53,7 +53,9 @@ class Storage {
         configText = decrypt(self.password, configText);
 
         try {
+          var configArray = [];
```

Comment:
configArray is an object so it should be declared as object and not as an array

let configArray = self.migrate(JSON.parse(cleanJsonString(configText)));